domains/casheditor.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "CookieTriste",
+           "email": "",
+           "discord": "614537315649323051",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.DyZqavAFOr4VsO9T465FPn5VkKEavuwmzXYfyQpUCSBcNwopWwuKTZNW0ZaJq9MSjp_64gqW48xD-ZvSbD86fIuWJUlDTtYNEtAXDoIU5mu65eviAjMkcIkgwMh-PioKXa-yYsT9ejaG_tBtEnZ570SUAA-K4hmIESY2965GQFt2vPS-JuNTeedhc9jj0CHb-UORBTTPdAGh6yrxXBDFnfi_sJF-bo9Lz10NGk00of9r0LSZ9F299m3QqMxt1lvkZhUlaNAmb8tFRyJUhq09w_XW7Dw2oYc-H1KyEBpCl2dmvxoysH3FDX5--VsrIamyB2OOnwnaaAlZVnD2HiT1UQ.-COjPlCc93AET-7GVf4plg.mvP2AiqbCqKNJ9gjpPos2WHbgiCCU35raAmR-sr7q-R53Z-Uxwz9Mh483BdPiy2xV1NvDqa3of3IEEL3Zi1n9BL8WWRtpuv99y_o7zG1g18.uTXovzwo8598DiSF0dRD2g"
+        },
+    
+        "record": {
+            "A": ["54.36.99.52"]
```

Comment:
404 Page. Can you send a preview of the site?

Reply:
Sorry forgot to start the according node.js instance. I got a domain name since then, but thanks for the help. Have a great day